pom.xml

```diff
@@ -18,8 +18,8 @@
     <name>Json Module</name>
 
     <properties>
-        <jackson.version>2.13.2</jackson.version>
-        <jackson.databind.version>2.13.2.1</jackson.databind.version>
+        <jackson.version>2.13.4</jackson.version>
+        <jackson.databind.version>2.13.4</jackson.databind.version>
```

Comment:
This version has a [vulnerability](https://mvnrepository.com/artifact/com.fasterxml.jackson.core/jackson-databind), isn't there a new PR from renovate to update it?